packages/create-llama/create-app.ts

```diff
@@ -87,21 +87,38 @@ export async function createApp({
     tools,
   };
 
+  let installationErrors = [];
+
   if (frontend) {
     // install backend
     const backendRoot = path.join(root, "backend");
     await makeDir(backendRoot);
-    await installTemplate({ ...args, root: backendRoot, backend: true });
+    try {
+      await installTemplate({
+        ...args,
+        root: backendRoot,
+        backend: true,
+      });
+    } catch (error) {
+      installationErrors.push(error);
+      console.log(red(`${error}`));
+    }
     // install frontend
     const frontendRoot = path.join(root, "frontend");
     await makeDir(frontendRoot);
-    await installTemplate({
-      ...args,
-      root: frontendRoot,
-      framework: "nextjs",
-      customApiPath: `http://localhost:${externalPort ?? 8000}/api/chat`,
-      backend: false,
-    });
+    try {
+      await installTemplate({
```

Comment:
what about the normal `installTemplate` case (frontend and backend are the same)?

Reply:
seems to me if it throws an error they are handled one level above - but we should have the same error handling

Reply:
in that case the `frontend` will be false then it'll go through other branch, see:
https://github.com/run-llama/LlamaIndexTS/blob/0ac403106743ccb8c1baa60e6fe47de4871f78b9/packages/create-llama/create-app.ts#L128

the app the will be the same as is